ci/docker/bosh-oracle-cpi-boshrelease/build-image.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+DOCKER_IMAGE=${DOCKER_IMAGE:-10.0.1.5:5000/boshcpi/oracle-cpi-release}
```

Comment:
Assumes a locally installed docker registry.  Can we not use dockerhub?  The credentials, repository and other info to push the image can come from the pipeline environment.

Reply:
Ok. I'll use my docker hub account for now. Eventually it needs to go here - https://hub.docker.com/r/boshcpi/

Reply:
Will use https://hub.docker.com/r/bitsarvi/oci-cpi-release/ for now

Reply:
I was hoping that we'd be able to accept docker credentials from the environment and use `docker login` followed by `docker push oci-cpi-release`, instead of using URIs with usernames.   Then the env is configured to use boshcpi or bitsarvi or whoever. Is that not doable?

Reply:
While the push part can be parameterized, the docker URI is also referred to in other files - e.g. https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/blob/master/ci/tasks/build-candidate.yml - that cannot contain variables that can be replaced/supplied at runtime. i'm confirming this on the concourse slack channel